### PR TITLE
[hotfix] [javadocs] Fix typo in org.apache.flink.sql.parser.ddl.SqlCreateTable

### PR DESCRIPTION
**Pull request by chenxyz707**
## What is the purpose of the change

Fix typo in org.apache.flink.sql.parser.ddl.SqlCreateTable

## Brief change log

*(for example:)*
  - Fix typo in org.apache.flink.sql.parser.ddl.SqlCreateTable

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: ( no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java [hunk 1]
```diff
@@ -49,14 +49,17 @@ public void pointTo(MemorySegment[] segments, int offset, int sizeInBytes) {
         this.sizeInBytes = sizeInBytes;
     }
 
+    @Override
```
Comment (MartijnVisser) [inline flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:52]: I don't think these changes are relevant to the hotfix doc PR?

File: flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java [hunk 2]
```diff
@@ -49,14 +49,17 @@ public void pointTo(MemorySegment[] segments, int offset, int sizeInBytes) {
...
     public MemorySegment[] getSegments() {
         return segments;
     }
 
+    @Override
```
Comment (MartijnVisser) [inline flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:57]: Same as above

File: flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java [hunk 3]
```diff
@@ -49,14 +49,17 @@ public void pointTo(MemorySegment[] segments, int offset, int sizeInBytes) {
...
     public int getOffset() {
         return offset;
     }
 
+    @Override
```
Comment (MartijnVisser) [inline flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:62]: Same as above

File: flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/collector/TableFunctionCollector.java [hunk 4]
```diff
@@ -77,6 +77,7 @@ public boolean isCollected() {
         return collected;
     }
 
+    @Override
```
Comment (MartijnVisser) [inline flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/collector/TableFunctionCollector.java:80]: Same as above

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
Thanks for your PR. I do see some changes that are not relevant to a typo fix, can you please take a look?

File: flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java (hunk 1 above)
Comment (chenxyz707) [reply flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:52]: These methods are missing @Override annotations.  I added

Comment (chenxyz707) [reply flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:52]: These require a new PR or do not need to be fixed ?

Comment (MartijnVisser) [reply flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java:52]: I'm not sure the methods are missing and I would remove them from a hotfix PR on documentation

Comment (chenxyz707) [reply flink-table/flink-table-common/src/main/java/org/apache/flink/table/data/binary/BinarySection.java]: The @Override annotation modifications have been removed in this pr

**Comment by MartijnVisser** [comment]
@chenxyz707 Thanks for that. Can you squash the commits on your end?

**Comment by chenxyz707** [comment]
> @chenxyz707 Thanks for that. Can you squash the commits on your end?

Commit and commit have been modified

**Review by MartijnVisser — APPROVED** [review]
Thanks you. LGTM
@fapaul Can you merge this?

**Review by fapaul — APPROVED** [review]
LGTM

**Pull request merged**